src/rnp/rnp.c

```diff
@@ -315,10 +347,12 @@ rnp_cmd(rnp_cfg_t *cfg, rnp_t *rnp, int cmd, char *f)
         ctx.sigcreate = get_birthtime(rnp_cfg_get(cfg, CFG_BIRTHTIME));
         ctx.sigexpire = get_duration(rnp_cfg_get(cfg, CFG_DURATION));
 
+        clearsign = (cmd == CMD_CLEARSIGN) ? true : false;
```

Comment:
I think you should also declare ``clearsign`` as ``bool``. Also 
``clearsign = (cmd == CMD_CLEARSIGN)``
means same thing with less code